sunpy/coordinates/transformations.py

```diff
@@ -41,6 +40,7 @@
     SphericalRepresentation,
     UnitSphericalRepresentation,
 )
+from astropy.coordinates.sky_coordinate.erfa import obl06
```

Comment:
I believe what you actually need to do is:
```suggestion
from astropy.coordinates.sky_coordinate import erfa
```
and then change the call on line 892 from `obl06()` to `erfa.obl06()`.